plugins/serendipity_plugin_syndication/serendipity_plugin_syndication.php

```diff
@@ -1,9 +1,5 @@
 <?php
 
-if (IN_serendipity !== true) {
```

Comment:
Rly, i should to turn it back. I didn't want to change these lines, it's mistake.